src/test/suite/fixtures/recorded/languages/latex/changeType.yml

```diff
@@ -0,0 +1,25 @@
+languageId: latex
```

Comment:
Huh.  Not obvious to me that this one would be "type".  I think in the future we'll support something like "callee", ie the function that is being called in a function call expression.  Could introduce something like that now and use it here?